plover/gui/main.py

```diff
@@ -73,9 +73,15 @@ class MainFrame(wx.Frame):
     DISCONNECTED_IMAGE_FILE = os.path.join(ASSETS_DIR, 'disconnected.png')
     REFRESH_IMAGE_FILE = os.path.join(ASSETS_DIR, 'refresh.png')
     BORDER = 5
-    RUNNING_MESSAGE = "running"
-    STOPPED_MESSAGE = "stopped"
-    ERROR_MESSAGE = "error"
+    STATUS_DISCONNECTED, STATUS_OUTPUT_ENABLED, STATUS_OUTPUT_DISABLED = range(3)
```

Comment:
I'd expect these to be disconnected => -1, disabled => 0, enabled => 1, but since this isn't exposed outside the app yet, no big deal.